test/dotnet-new3.UnitTests/DotnetNewSearch.cs

```diff
@@ -871,7 +871,7 @@ private List<List<string>> ParseTableOutput(string stdOut, string[] expectedColu
             int headerLineIndex = Array.FindIndex(lines, line => expectedColumns.All(column => line.Contains(column)));
             string headerLine = lines[headerLineIndex];
             //table ends after empty line
-            int lastLineIndex = Array.FindIndex(lines, headerLineIndex + 1, line => line.Length == 0) - 1;
+            int lastLineIndex = Array.FindIndex(lines, headerLineIndex + 1, line => (line.Length == 0 || line.Contains("[Debug]"))) - 1;
```

Comment:
What is this change for?

Reply:
this is for https://github.com/dotnet/templating/pull/4134

this commit will vanish after rebase to main.